laravel-app/resources/views/profile/show.blade.php

```diff
@@ -8,7 +8,7 @@
 	<div class="p-3">
 		<div class="row justify-content-center">
 			<div class="col-md-4 text-center">
-				<img class="img-fluid w-50" src="{{asset($imagePath) }}" alt="Microblog Logo">
+				<img class="img-fluid w-50" src="{{ asset('storage/'. $user->image_path) }}" alt="Microblog Logo">
```

Comment:
same with this. so that you would not to add the 'storage/'

Reply:
added storage to file path in attribute sir, thank you ❤️